src/app.module.ts

```diff
@@ -49,14 +49,17 @@ export class AppModule implements NestModule {
   // into account. The .env file loading is done by the ConfigurationModule
   // which is not available at this stage.
   static register(configFactory: ConfigFactory = configuration): DynamicModule {
+    const isAlertsFeatureEnabled = configFactory()['features']['alerts'];
     const isEmailFeatureEnabled = configFactory()['features']['email'];
 
     return {
       module: AppModule,
       imports: [
         // features
         AboutModule,
-        AlertsModule,
+        ...(isAlertsFeatureEnabled && isEmailFeatureEnabled
```

Comment:
This PR is doing more things than described I would say.

- Extracting a Alerts Domain Module shouldn't change how the feature is loaded on the application.
- The way this is bundled here raises the question of why we need two feature flags instead of one.

Reply:
I agree, I've changed both the PR description and its code. Now it only extracts the Alerts controller/domain modules. I'll be doing the other changes in separate PRs. Thanks.